.github/workflows/docusaurus.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v3
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          deploy_key: ${{ secrets.GH_PAGES_DEPLOY_KEY }}
```

Comment:
I think the key comes from the action and should stay the same, while you should change the env var as @nwmac suggested.

```yml
          github_token: ${{ secrets.GH_PAGES_DEPLOY_KEY }}
```

Reply:
@cnotv No, this is wrong - we are using a github deployment key and not a github token - the former allows us to have a key that only has write access to the dashboard repo. This is a different kind of access, so the key needs to be `deploy_key` - see:  https://github.com/peaceiris/actions-gh-pages#supported-tokens

Reply:
Oh ok sorry about it, I did not understand that were equivalent.